website/routes.py

```diff
@@ -197,6 +198,13 @@ def make_url_map(app):
             OsfWebRenderer('public/pages/meeting_landing.mako'),
         ),
 
+        Rule(
+            '/api/v1/citation_styles/',
```

Comment:
I know this is still v1 api, but for new routes, we should try to match the eventual upcoming scheme. Could this be '/api/v1/citations/styles/' instead?